CMP1903_A1_2324/Die.cs

```diff
@@ -42,7 +42,7 @@ public void Roll()
                 * Due to random changing every clock tick a pause timer is required, 
                 * so before calling random a timer of 200 miliseconds is called to allow a new random number tyo be generated
                 */
-                int millisecond = 200;
+                int millisecond = 500;
                 Thread.Sleep(millisecond);
```

Comment:
creating an int for a set delay is unnecessary you can just put 1 in the brackets of thread.sleep to have the same effect